api/index.js

```diff
@@ -117,10 +119,11 @@ Ekke.commands = {
  * @public
  */
 Ekke.defaults = {
-  'hostname': 'localhost',    // Hostname we should create our server upon
-  'port': 1975,               // The port number of the created server
-  'silent': true,             // Silence Metro bundler
-  'reset-cache': true         // The cache key poorly handled, so turn off by default.
+  'hostname': 'localhost',                                 // Hostname we should create our server upon
+  'port': 1975,                                            // The port number of the created server
+  'silent': true,                                          // Silence Metro bundler
+  'reset-cache': false,                                    // Turn off the Metro bundler cache
```

Comment:
The current functionality was actually Working As Intended (tm), the `argh` library we use to parse the CLI flags allows flags to be prefixed with `no-` or `disable-` to set the flag the false.

As the caching metro bundle is causing issues quite often I opted to disable this by default by setting the `reset-cache` flag here to true. So to enable caching you need to use `--no-reset-cache` to set the option to false.

The question however is, do we want to continue deleting the cache on each start of `ekke` and keep this change.

Reply:
Thanks for the writeup, that explains the `no-reset-cache`! :)

I leave the decision up to you. I don't have a very strong opinion here, just let me know if you want to put the old cache clearing behaviour back. :)